symfony/security-bundle/5.3/config/packages/security.yaml

```diff
@@ -25,3 +25,16 @@ security:
     access_control:
         # - { path: ^/admin, roles: ROLE_ADMIN }
         # - { path: ^/profile, roles: ROLE_USER }
+
+when@test:
```

Comment:
Can we use `when@test` here? It's probably fine... but there's no guarantee the user has FWBundle 5.3 :/

Reply:
that's not related to FrameworkBundle. That's related to the version of the DI component and of HttpKernel. And for those, we have that guarantee